specification/trace/semantic_conventions/http.md

```diff
@@ -194,6 +195,22 @@ Following attributes MUST be provided **at span creation time** (when provided a
 
 Note that in some cases host and port identifiers in the `Host` header might be different from the `server.address` and `server.port`, in this case instrumentation MAY populate `Host` header on `http.request.header.host` attribute even if it's not enabled by user.
 
+### HTTP client span duration
+
+There are some minimal constraints that SHOULD be honored:
+
+* HTTP client spans SHOULD start sometime before the first request byte is sent. This may or may not include connection time.
+* HTTP client spans SHOULD end be sometime after the HTTP response headers are fully read (or when they fail to be read). This may or may not include reading the response body.
```

Comment:
```suggestion
* HTTP client spans SHOULD end sometime after the HTTP response headers are fully read (or when they fail to be read). This may or may not include reading the response body.
```